src/main/java/StringCalculator/StringCalculator.java

```diff
@@ -0,0 +1,74 @@
+package StringCalculator;
+
+import java.util.Arrays;
+import java.util.Deque;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Queue;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+    private static final Pattern CHECK_NUMBER_PATTERN = Pattern.compile("-?\\d+(\\.\\d+)?");
+
+    public static int calculate(String input) {
+        validate(input);
+        List<String> strings = Arrays.asList(input.split(" "));
```

Comment:
```suggestion
        List<Node> nodes = Arrays.asList(input.split(" "));
        // 또는
        Nodes nodes = new Nodes(input, " ");
```

객체로 도전 해보았어도 좋았을 것 같아요! ㅎㅎ

> 적용은 안하셔도 됩니다~ 관점 공유입니다~